src/controller_store.coffee

```diff
@@ -0,0 +1,28 @@
+camelCase = require 'camel-case'
+FileIdentifier = require './file_identifier'
+fs = require 'fs'
+path = require 'path'
+
+
+class ControllerStore
+
+  constructor: (@controllersDir, @dependencies) ->
+    @controllers = {}
+
+
+  # Calls a controller action with args
+  applyAction: ({controllerName, actionName, args}) ->
+    @controllers[camelCase controllerName][actionName] args...
+
+
+  # Populates @controllers by instantiating controllers found in @controllersDir
+  initialize: ->
+    for fileName in fs.readdirSync @controllersDir
```

Comment:
The thing it blocks is all other parts of the app initializing and booting up, thereby leading to increased startup times of any server that uses exprestive. That's bad imho.

Looking forward to seeing how you'll block HTTP requests!